swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -90,27 +78,7 @@
 			"in": "query",
 			"name": "if",
 			"type": "string",
-			"enum": [
-				"oic.if.baseline",
-				"oic.if.r"
-			]
-		},
-		"events-filtered-by-di": {
-			"in": "query",
-			"name": "di",
-			"required": false,
-			"$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.types-schema.json#/definitions/uuid",
-			"x-example": "fdaa4ef4-e213-4268-b336-14c285533c01"
-		},
-		"events-filtered-by-category": {
-			"in": "query",
-			"name": "categoryfilter",
-			"required": false,
-			"type": "integer",
-			"enum": [
-				1, 2, 4, 8, 16, 32, 64
-			],
-			"x-example": 7
+			"enum": ["oic.if.baseline"]
```

Comment:
Was the agreement with the Sec WG just to remove oic.if.r rather than changing it to oic.if.rw?  Per the Core Spec, oic.if.baseline is not to be used for UPDATE so to comply with the spec, we should add oic.if.rw.

Reply:
I double-checked the meeting minutes, and couldn't find anything related to the oic.if.r
Leaving just baseline makes it consistent with other SVRs.